boilerplate/app/screens/DemoPodcastListScreen.tsx

```diff
@@ -57,6 +58,37 @@ export const DemoPodcastListScreen = observer(function DemoPodcastListScreen(
         contentContainerStyle={$flatListContentContainer}
         refreshing={refreshing}
         onRefresh={manualRefresh}
+        ListEmptyComponent={
+          <View>
+            <View style={$container}>
+              {!episodeStore.favoritesOnly ? (
+                <View>
+                  <Text preset="subheading" style={$subheading}>
+                    So empty... so sad
+                  </Text>
+                  <Text>No data found yet. You can click the button or reload the app.</Text>
+                </View>
+              ) : (
+                <View>
+                  <Text preset="subheading" style={$subheading}>
+                    You don't have any favorites yet
+                  </Text>
+                  <Text>Please click the heart icon on each list to add it to your favorites.</Text>
+                </View>
```

Comment:
These texts are not added to i18n since they will probably be changed when the actual design is ready.